.obs/dockerfile/SL-Micro-base-container/_service

```diff
@@ -11,4 +11,14 @@
     <param name="eval">IMG_REPO=$(rpm --macros=/root/.rpmmacros -E %img_repo)</param>
     <param name="var">IMG_REPO</param>
   </service>
+  <service name="replace_using_env" mode="buildtime">
+    <param name="file">Dockerfile.iso</param>
+    <param name="eval">SLMICRO_VERSION=$(rpm --macros=/root/.rpmmacros -E %slmicro_version)</param>
+    <param name="var">SLMICRO_VERSION</param>
+  </service>
+  <service name="replace_using_env" mode="buildtime">
```

Comment:
nit: we could consider adapting this `replace_using_env` OBS service to actually support globbing, so instead of requiring a new service call per file and per variable we could do something like the following (the `*` on the file name):

```xml
<service name="replace_using_env" mode="buildtime">
    <param name="file">Dockerfile*</param>
    <param name="eval">SLMICRO_VERSION=$(rpm --macros=/root/.rpmmacros -E %slmicro_version)</param>
    <param name="var">SLMICRO_VERSION</param>
  </service>
```